include/device_config.h

```diff
@@ -28,17 +28,21 @@
 // #define HAS_BALANCE_BOARD_INTEGRATION
 
 // Enable the yukkuri voice talking clock
+#ifndef WITHOUT_AQUESTALK
 #define HAS_AQUESTALK
+#endif
 
 // Disable the faux brightness reduction for some UI elements by drawing them only every other frame
 // #define COMPOSABLE_NO_EVENODD
 
 // ---- HARDWARE
 
+#ifndef WITHOUT_BLUETHOOTH
```

Comment:
As long as the Bluetooth features (WoSensor or Balance Board) are not enabled the code isn't included in the binary so I think this is unnecessary